tests/unit/NetworkTest.js

```diff
@@ -168,17 +169,17 @@ describe('NetworkTests', () => {
                 // And then make 3 API requests in quick succession with an expired authToken and handle the response
                 // It doesn't matter which requests these are really as all the response is mocked we just want to see
                 // that we get re-authenticated
-                App.openApp();
-                App.openApp();
-                App.openApp();
+                PersonalDetails.openPersonalDetails();
```

Comment:
I had to use a READ command here to make the test pass. It seems like this behaves differently when we call a write command.

Reply:
I tested this scenario though of a write command with an expired authToken and I was re-authenticated properly.

Reply:
On it.